### PR TITLE
Add centosOvs() that install.sh can build rpm packages

### PR DESCRIPTION
**Pull request by pichuang**

### CODE REVIEW

File: util/install.sh [hunk 1]
```diff
@@ -783,7 +828,11 @@ else
       t)    vm_other;;
       v)    ovs;;
       V)    OVS_RELEASE=$OPTARG;
-            ubuntuOvs;;
+            case $DIST in
+            Ubuntu|Debain) ubuntuOvs;;
```
Comment (lantz) [inline util/install.sh]: Debain?

Comment (pichuang) [inline util/install.sh]: Sorry, just a typo. XD

**Comment by lantz** [comment]
I think I prefer Gregory Gee's build commands - they're more concise and easier to follow.

http://techandtrains.com/2014/09/13/mininet-and-open-vswitch-on-centos-7/

**Comment by pichuang** [comment]
kernel datapath work fine for me
centos7_openvswitch

File: util/install.sh [hunk 2]
```diff
@@ -328,6 +329,56 @@ function ubuntuOvs {
...
+    # Get rpm build deps
+    $install rpm-build redhat-rpm-config openssl-devel gcc gcc-c++ make wget
+
+    # Prepare rpmbuild enviroment
+    mkdir -p $OVS_SRC/rpmbuild/SOURCES/ && cd $OVS_SRC/rpmbuild/SOURCES/
+    mv $OVS_SRC/openvswitch-$OVS_RELEASE* $OVS_SRC/rpmbuild/SOURCES/
+
+    # Build OVS
+    cd $OVS_SRC/rpmbuild/SOURCES/openvswitch-$OVS_RELEASE
+    sudo rpmbuild -bb --define "_topdir $OVS_SRC/rpmbuild" --without check rhel/openvswitch.spec
```
Comment (pichuang) [inline util/install.sh]: Follow your suggest about build path, is this good?

**Pull request closed without merging**